libs/platform/src/lib/components/form/form-group/form-field/form-field.component.html

```diff
@@ -3,20 +3,22 @@
 </ng-template>
 
 <ng-template #renderer>
-    <div [horizontal]="labelLayout === 'horizontal'" fd-form-item>
-        <label [attr.for]="id"
-               [required]="editable && required"
-               fd-form-label
-               [inlineHelpTitle]="hint"
-               [inlineHelpBodyPlacement]="hintPlacement">
-            <span *ngIf="!noLabelLayout">{{ label }}</span>
-        </label>
+    <div [horizontal]="labelLayout === 'horizontal'" fd-form-item class="fd-row">
+        <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
```

Comment:
![image](https://user-images.githubusercontent.com/26483208/104178142-227dad80-540a-11eb-8654-5aa53d0bb4b1.png)

That's how labels look on md (>600px && <1024px). Is it correct?

Reply:
According to styles the behavior is correct